crates/astria-sequencer/src/utils.rs

```diff
@@ -16,9 +20,29 @@ impl<'a> std::fmt::Display for Hex<'a> {
     }
 }
 
+pub(crate) fn anyhow_to_eyre(anyhow_error: anyhow::Error) -> astria_eyre::eyre::Report {
+    let anyhow_result = Err::<(), _>(anyhow_error);
+    let boxed: Result<(), Box<dyn std::error::Error + Send + Sync>> =
+        anyhow_result.map_err(std::convert::Into::into);
+    let Err(err) = boxed else {
+        panic!("anyhow_to_eyre called on `Ok`")
+    };
```

Comment:
This err -> result -> err dance is not necessary:
```suggestion
    let boxed: Box<dyn std::error::Error + Send + Sync> = anyhow::Error.into();
    eyre!(boxed)
```